bench/setup_bench.sh

```diff
@@ -9,10 +9,10 @@ trap "kill 0" EXIT
 
 cifar_path=$1
 
-python bench_init.py $cifar_path
+python bench/bench_init.py $cifar_path
 
 export CLIPPER_MODEL_NAME="bench_sklearn_cifar"
 export CLIPPER_MODEL_VERSION="1"
 export CLIPPER_MODEL_PATH="model/"
 
-python ../containers/python/sklearn_cifar_container.py 
+python containers/python/sklearn_cifar_container.py 
```

Comment:
Changes to this file were made so that the usage specified in the end_to_end_bench script, namely that setup_bench.sh should be run from the root clipper folder, works. Before it worked when run from the /bench folder.